tsconfig.json

```diff
@@ -1,21 +1,28 @@
 {
-  "extends": "./.tsconfig-paths.json",
-  "compilerOptions": {
-    "target": "ESNext",
-    "module": "ESNext",
-    "moduleResolution": "bundler",
-    "jsx": "preserve",
-    "jsxImportSource": "vue",
-    "strict": true,
-    "noEmit": true,
-    "noImplicitThis": false,
-    "verbatimModuleSyntax": true,
-    "useDefineForClassFields": true,
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "resolveJsonModule": true,
-    "skipLibCheck": true,
-    "types": ["vitest/globals"],
-    "allowImportingTsExtensions": true
-  }
-}
+    "extends": "./frontend_configuration/tsconfig-paths.json",
```

Comment:
These are the changes I'm less sure about. I know we don't need to support runserver'ing arches-component-lab, esp. on 7.6, but I don't want cryptic typescript failures on 7.6 for projects using it. Has this been tested?

Reply:
nah this is fine, this file is only used when running arches-component-lab as a project, not an app.

Reply:
Great, just checking. Tested in provenance and nothing seems amiss